data/data/aws/route53/base.tf

```diff
@@ -6,11 +6,13 @@ locals {
   public_zone_id = "${data.aws_route53_zone.base.zone_id}"
 
   zone_id = "${var.private_zone_id}"
+
+  cluster_domain = "${var.cluster_name}.${var.base_domain}"
```

Comment:
This package could be adjusted to take the unsplit cluster domain if we replaced [this][1] with Go logic to snip subdomains off the full cluster name until we found a public zone.  Or we could restructure the install-config to make the public zone's domain part of the AWS-specific platform configuration.  Both of those are probably more work than we want to sink into initial exploration, but I thought I'd file a note to track future polish possibilities ;).

[1]: https://github.com/openshift/installer/blob/v0.11.0/data/data/aws/route53/base.tf#L1-L3